app/views/users/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if user_signed_in? %>
+  <%= button_to 'Sign out', destroy_user_session_path(@user), method: :delete, data: { turbo: false }, class: "" %>
```

Comment:
[optional] Please remove the sign out button as this has been included to the navbar